components/ProfileMenu.vue

```diff
@@ -261,15 +255,13 @@ const closeContentLanguageDialog = () => {
           <div
             class="text-label-1 bg-background-1 min-w-[100px] px-2 py-1.5 rounded-lg font-semibold shadow-[0_4px_12px_0_#0000000D,0_1px_4px_0_#0000000D,0_0_0_1px_#0000000D]"
           >
-            <select v-model="$i18n.locale" class="py-1">
+            <select v-model="profileStore.uiLanguage" class="py-1">
               <option
                 v-for="(lang, i) in $i18n.availableLocales"
                 :key="`Lang${i}`"
                 :value="lang"
               >
-                {{
-                  `${languageDictionary[lang]?.NativeName} (${languageDictionary[lang]?.EnglishName})`
-                }}
+                {{ getLocalizedLanguageName(lang) }}
```

Comment:
We should still show the native name in brackets

Reply:
Always, or just if it's not translated?

Reply:
Always. Imagine that you selected Russian. Then you have no idea which of the 25 listed languages is english or german. Therefore it makes a lot of sense to show the native name as well, when selecting a language.

Reply:
Should we unify this approach, or where should it differ? Should it be only different for showing the ui language?

Approach: For this I'd extend the function `getLocalizedLanguageName()` to take a second argument which is the language. If not provided, it takes the default, which is the one selected as ui language.

Reply:
In Figma, native name in brackets is used for UI language and content language.